packages/sakuli-legacy/src/context/common/actions/screen.function.ts

```diff
@@ -37,7 +37,7 @@ export const getTimestamp = (
   return timestampString.split(":").join("-");
 };
 
-export type SearchResult = {
+export type ScreenRegionResult = {
```

Comment:
Actually no, since the `ScreenRegionResult` is only used internally and the user-facing API is returning a `Region`

```typescript
const focusedWindowRegion = await ScreenApi.getRegionFromFocusedWindow();
const RegionImpl = createRegionClass(ctx, project);
return new RegionImpl(
      focusedWindowRegion.left,
      focusedWindowRegion.top,
      focusedWindowRegion.width,
      focusedWindowRegion.height
);
```

We could however get rid of this intermediate representation by providing the current `TestExecutionContext` and `Project` to the `ScreenApi` so we're able to directly instantiate a new `Region`

Reply:
Ah, now I see! :+1: